numpy/core/src/common/simd/vsx/arithmetic.h

```diff
@@ -94,10 +94,25 @@
 #define npyv_mul_f32 vec_mul
 #define npyv_mul_f64 vec_mul
 
+#define npyv_muladd_f32 vec_madd
+#define npyv_muladd_f64 vec_madd
+
 /***************************
  * Division
  ***************************/
 #define npyv_div_f32 vec_div
 #define npyv_div_f64 vec_div
 
+// TODO: Horizontal add: Calculates the sum of all vector elements.
```

Comment:
Assuming I read this right, perhaps better as:
```suggestion
// Horizontal add: Calculates the sum of all vector elements.
// TODO: optimize this
```

Reply:
Sorry, this TODO needs to delete, I didn't find a more efficient way to do the sum in VSX right now.